### PR TITLE
update readme with extensions and fileGlob

### PR DESCRIPTION
**Pull request by collingreen**
- add `extensions` description in options section
- add `fileGlob` description in options section
- add 'Tips and Tricks' section with `extensions` and `fileGlob` usage

### CODE REVIEW

**Comment by collingreen** [comment]
@jnordberg: what is your preference on adding examples? I have two additional example config files that demonstrate using extensions and fileGlob - would that be helpful to include in this PR?

File: README.md [hunk 1]
```diff
@@ -73,6 +75,34 @@ A project with a heavy dependency can impact bundle times, this example uses req
 ```
 
 
+Tips and Tricks
+---------------
+
+[Sometimes](https://github.com/jnordberg/wintersmith-browserify/issues/3) you only want to browserify specific files or folders instead of all of a particular file type. You can control exactly which files get passed to browserify with either the `extensions` option or the `fileGlob` option (but not both - `fileGlob` overwrites `extensions`).
```
Comment (jnordberg) [inline README.md]: We should point out that the `extensions` option does more than just set the build the fileGlob pattern. They are also passed to browserify so you can `require('./foo')` and have it resolve `foo.coffee` for example.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ In addition to browserify's standard options wintersmith-browserify adds the fol
   * `requires` - `[{"filename": ["module", {name: "module", "expose": "exposed_name"}, ..]}, ..]` - per-file bundle.require() mapping
   * `externals` - `[{"filename": ["module", ..]}, ..]` - per-file bundle.external() mapping
   * `static` - `["filename", ..]` - list of files that will only be compiled once and cached in memory for subsequent requests
+  * `extensions` - `[".ext", ..]` - list of file extensions for matching files - default: ['.js', '.coffee']
```
Comment (jnordberg) [inline README.md]: See other comment

**Comment by jnordberg** [comment]
Looks good, just need to clarify the extensions option a bit.

Feel free to add examples, you can move `example` to `examples/basic` and add your other examples in that dir

**Comment by collingreen** [comment]
@jnordberg - updated the extensions info a bit. If you have suggestions on how to word it better, just let me know. 

Also added two new example folders and a short example readme.

**Comment by jnordberg** [comment]
LGTM, merging

Thanks for contributing!

**Comment by jnordberg** [comment]
Published new version with the new option now as 0.8.0

**Comment by collingreen** [comment]
Thanks!

**Pull request merged**